config/jest.config.js

```diff
@@ -0,0 +1,39 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  clearMocks: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.{js,jsx}', '!<rootDir>/src/index.jsx'],
+  coverageDirectory: '<rootDir>/coverage',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Non-HTML variable 'coverageDirectory' is used to store raw HTML](https://app.codacy.com/manual/iagoregiani/verificador-associacao/pullRequest?prid=5497764)